sql/log_event.cc

```diff
@@ -10500,6 +10500,30 @@ int Rows_log_event::handle_idempotent_and_ignored_errors(Relay_log_info const *r
                            (rbr_exec_mode == RBR_EXEC_MODE_IDEMPOTENT));
     bool ignored_error= (idempotent_error == 0 ?
                          ignored_error_code(actual_error) : 0);
+#if defined(WITH_WSREP)
```

Comment:
Please change it to `#ifdef WITH_WSREP` in order to make it work with our `check_src` tool.